src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -289,3 +289,31 @@
 	margin-left: 4px;
 	color: inherit;
 }
+
+.monaco-workbench .pane-body.integrated-terminal .tabs-container.has-text .tabs-list .terminal-tabs-entry .uri-icon {
+	background-repeat: no-repeat;
+	background-position: left center;
```

Comment:
Did center center not work? Also this may be redundant with `background-size: contain`?